src/Core/Managed/Wp80/Extensibility/Implementation/TelemetryConfigurationFactory.cs

```diff
@@ -50,6 +50,12 @@ public virtual void Initialize(TelemetryConfiguration configuration)
 
             // Creating the default channel if no channel configuration supplied
             configuration.TelemetryChannel = configuration.TelemetryChannel ?? new InMemoryChannel();
+
+            // Creating the the processor chain with default processor (transmissionprocessor) if none configured            
+            if (configuration.TelemetryProcessorChain == null)
+            {
+                new TelemetryProcessorChainBuilder().Build(configuration);
```

Comment:
Ideally we should not change this file, since right now TelemetryProcessors are not for wp8